lnwire/lnwire_test.go

```diff
@@ -65,6 +73,19 @@ func randPartialSig(r *rand.Rand) (*PartialSig, error) {
 	}, nil
 }
 
+func somePartialSig(t *testing.T, r *rand.Rand,
+) tlv.OptionalRecordT[PartialSigType, PartialSig] {
```

Comment:
formatting nit: 

```
func somePartialSig(t *testing.T,
	r *rand.Rand) tlv.OptionalRecordT[PartialSigType, PartialSig] {
```

same above

Reply:
By above do you mean `sameLocalNonce`? Don't see how it can be folded like this, unless we break before the first arg `r` which looks even funkier IMO.